spinegeneric/cli/package_for_correction.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+#
+# Script to package data for manual correction.
+#
+# For usage, type: python package_for_correction.py -h
+#
+# Author: Julien Cohen-Adad
+
+
+import os
+import sys
+import shutil
+import tempfile
+from textwrap import dedent
+import argparse
+import yaml
+import coloredlogs
+
+import spinegeneric as sg
+import spinegeneric.utils
+import spinegeneric.bids
+
+
+def get_parser():
+    """
+    parser function
+    """
+    parser = argparse.ArgumentParser(
+        description='Package data for manual correction. In case processing is ran on a remote cluster, it it '
+                    'convenient to generate a package of the files that need correction to be able to only copy these '
+                    'files locally, instead of copying the ~20GB of total processed files.',
+        formatter_class=sg.utils.SmartFormatter,
+        prog=os.path.basename(__file__).strip('.py')
+    )
+    parser.add_argument(
+        '-config',
+        metavar=sg.utils.Metavar.file,
+        required=True,
+        help=
+        "R|Config yaml file listing images that require manual corrections for segmentation and vertebral "
+        "labeling. 'FILES_SEG' lists images associated with spinal cord segmentation, 'FILES_GMSEG' lists images "
+        "associated with gray matter segmentation and 'FILES_LABEL' lists images associated with vertebral labeling. "
+        "You can validate your yaml file at this website: http://www.yamllint.com/. Below is an example yaml file:\n"
+        + dedent(
+            """
+            FILES_SEG:
+            - sub-amu01_T1w_RPI_r.nii.gz
+            - sub-amu01_T2w_RPI_r.nii.gz
+            - sub-cardiff02_dwi_moco_dwi_mean.nii.gz
+            FILES_GMSEG:
+            - sub-amu01_T2star_rms.nii.gz
+            FILES_LABEL:
+            - sub-amu01_T1w_RPI_r.nii.gz
+            - sub-amu02_T1w_RPI_r.nii.gz\n
+            """)
+    )
+    parser.add_argument(
+        '-path-in',
+        metavar=sg.utils.Metavar.folder,
+        help='Path to the processed data. Example: ~/spine-generic/results/data',
+        default='./'
+    )
+    parser.add_argument(
+        '-o',
+        metavar=sg.utils.Metavar.folder,
+        help="Zip file that contains the packaged data. Default: data_to_correct.zip",
+        default='data_to_correct.zip'
```

Comment:
I had to delete suffix `.zip` at this line (to` default='data_to_correct'`). Otherwise, output file is named `some_name.zip.zip` because line 139 add zip suffix as well.

Reply:
great catch! fixed in 01069533b7dcf44211faa356bb0e7833713b1a55